web/scss/pages/_ancillary.scss

```diff
@@ -8,17 +8,45 @@
   h2,
   h3,
   h4,
-  h5,
+  h5 {
+    + blockquote,
+    + hr,
+    + iframe,
+    + img,
+    + pre,
+    + video,
+    + .button {
+      margin-top: 0.25rem;
+    }
+
+    + footer,
+    + form,
+    + h1,
+    + h2,
+    + h3,
+    + h4,
+    + h5,
+    + ol,
+    + p,
+    + ul {
+      padding-top: 0.25rem;
+    }
+
+    + .button {
+      display: inline-block;
```

Comment:
This is a bad rule... is this just to make the padding/margin above work?

If this is just to fix visual spacing for a button that immediately follows a header, it seems fine to wrap it in a `<p>` or a `<div>`

Reply:
I don't remember where this button is.